drivers/display/display_sdl.c

```diff
@@ -79,9 +79,10 @@ static int sdl_display_init(const struct device *dev)
 	}
 
 	int rc = sdl_display_init_bottom(config->height, config->width, sdl_display_zoom_pct,
-					 use_accelerator, &disp_data->window, &disp_data->renderer,
-					 &disp_data->mutex, &disp_data->texture,
-					 &disp_data->read_texture, &disp_data->background_texture,
+					 use_accelerator, &disp_data->window, dev,
+					 &disp_data->renderer, &disp_data->mutex,
+					 &disp_data->texture, &disp_data->read_texture,
+					 &disp_data->background_texture,
 					 CONFIG_SDL_DISPLAY_TRANSPARENCY_GRID_CELL_COLOR_1,
 					 CONFIG_SDL_DISPLAY_TRANSPARENCY_GRID_CELL_COLOR_2,
 					 CONFIG_SDL_DISPLAY_TRANSPARENCY_GRID_CELL_SIZE);
```

Comment:
Would turning `disp_data` into an argument to `sdl_display_init_bottom()` facilitate things?
I am still new to display and SDL, so am asking out of ignorance rather than experience.

Reply:
As far as I understand the code and also some previous questions towards @aescolar it is written on purpose like this so that there is no Zephyr depedency inside the "bottom" code and vice versa.

Reply:
Thank you for showing me the way around!